distribution/src/bin-regular/common.sh

```diff
@@ -39,12 +39,16 @@ if [ "$JAVA_VERSION" -ge "9" ]; then
     JDK_OPTS="\
         --add-modules java.se \
         --add-exports java.base/jdk.internal.ref=ALL-UNNAMED \
-        --add-exports jdk.management/com.ibm.lang.management.internal=ALL-UNNAMED \
         --add-opens java.base/java.lang=ALL-UNNAMED \
         --add-opens java.base/sun.nio.ch=ALL-UNNAMED \
         --add-opens java.management/sun.management=ALL-UNNAMED \
         --add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED \
     "
+
+    VM_NAME=$(${JAVA} -XshowSettings:properties -version 2>&1 | grep java.vm.name | cut -d "=" -f2)
```

Comment:
nit: we don't want to use local for compatibility with old version of bash?

Reply:
What version would _old version of bash_ would be? I am not familiar with `local` keyword?

Reply:
local reduces the visibility of variable in the script. Recent bash and zsh are supporting it (IIRC bash 4.1 was not supporting fully, like local -n), but it's not posix-standard thing. Nice if you can force / expect bash version ;)